manifests/init.pp

```diff
@@ -99,32 +114,34 @@
-  Variant[Enum['yes','no'],Boolean] $gzip_dbout        = 'yes',
-  Stdlib::Compat::Integer           $verbose           = '5',
-  Array[String]                     $report_urls       = [ 'file:@@{LOGDIR}/aide.report'],
-  Stdlib::Absolutepath              $ruledir           = '/etc/aide.conf.d',
-  Variant[Hash,Array[String]]       $rules             = {},
-  Boolean                           $enable            = false,
-  Simplib::Cron::Minute             $minute            = 22,
-  Simplib::Cron::Hour               $hour              = 4,
-  Simplib::Cron::Monthday           $monthday          = '*',
-  Simplib::Cron::Month              $month             = '*',
-  Simplib::Cron::Weekday            $weekday           = 0,
-  String                            $cron_command      = '/bin/nice -n 19 /usr/sbin/aide -C',
-  Boolean                           $logrotate         = simplib::lookup('simp_options::logrotate', { 'default_value' => false}),
-  Aide::Rotateperiod                $rotate_period     = 'weekly',
-  Integer                           $rotate_number     = 4,
-  Boolean                           $syslog            = simplib::lookup('simp_options::syslog', { 'default_value' => false }),
-  Aide::SyslogFacility              $syslog_facility   = 'LOG_LOCAL6',
-  Boolean                           $auditd            = simplib::lookup('simp_options::auditd', { 'default_value' => false }),
-  Integer                           $aide_init_timeout = 300,
-  String                            $package_ensure    = simplib::lookup('simp_options::package_ensure', { 'default_value' => 'installed' }),
+  Array[String]                            $aliases,
+  Variant[Array[String[1]],String]         $default_rules,
+  Stdlib::Absolutepath                     $dbdir             = '/var/lib/aide',
+  Stdlib::Absolutepath                     $logdir            = '/var/log/aide',
+  String                                   $database_name     = 'aide.db.gz',
+  String                                   $database_out_name = 'aide.db.new.gz',
+  Variant[Enum['yes','no'],Boolean]        $gzip_dbout        = 'yes',
+  Stdlib::Compat::Integer                  $verbose           = '5',
+  Array[String]                            $report_urls       = [ 'file:@@{LOGDIR}/aide.report'],
+  Stdlib::Absolutepath                     $ruledir           = '/etc/aide.conf.d',
+  Variant[Hash,Array[String]]              $rules             = {},
+  Boolean                                  $enable            = false,
+  Optional[Simplib::Cron::Minute]          $minute            = undef,
+  Optional[Simplib::Cron::Hour]            $hour              = undef,
+  Optional[Simplib::Cron::Monthday]        $monthday          = undef,
+  Optional[Simplib::Cron::Month]           $month             = undef,
+  Optional[Simplib::Cron::Weekday]         $weekday           = undef,
+  Optional[Enum['root', 'etc', 'systemd']] $cron_method       = undef,
```

Comment:
Which cron method is being selected?  Not clear here. Without a default, seems like no cron of any type will be run.